### ISSUE TITLE
Tree list items, refresh and more button is not accessible via keyboard tab key 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/monaco-editor#2753]
**GitHubTags:**#A11yMAS;#A11yTCS;#A11ySev1;#Keyboard;#Desktopweb;#WCAG2.1.1;#A11yAuto;#Win10;#AILimited;#TabStop;#Benchmark;#FTP;#Visual Studio Code Client;#VisualStudioCode-Web-Apr2021;#a11y-monacoeditor;

**Environment Details:**
#URL: https://ms.portal.azure.com/#home
Chromium edge: Version 95.0.1020.44 (Official build) (64-bit)

**Repro Steps:**

1. Open URL https://ms.portal.azure.com/
2. Navigate to the cloud shell button on the top menu bar and select it.  
3. Navigate to "Open Editor" button of "CloudShell-PowerShell Editor" window and select it
4. press tab to move to the editor and then, press tab and ctrl+M
5. use shift +tab focus goes on the entire tree list but not individual items

**Actual:**
Tree views, refresh and more button is not accessible via keyboard tab key. on shift+tab to the tree list the arrow keys do not move to the items.

**Expected Result:**
Tree views, refresh and more button should accessible via keyboard tab and arrow keys 

**User Impact:**
keyboard users will not be able to access the controls

**Attachment:**
![Presentation1](https://user-images.githubusercontent.com/24580570/140724690-c3407f7d-b714-468e-be1d-1e9f535907a0.gif)

**Comment by hediet** [comment]
Is this a monaco issue?

**Comment by fsteffi** [comment]
@hediet yes it is

**Comment by hediet** [comment]
I cannot access that page to reproduce:

![image](https://user-images.githubusercontent.com/2931520/140733098-508c3ff4-5839-4370-89fe-aa4bbdc20632.png)

**Comment by fsteffi** [comment]
closing this bug as per comment above "Please note that "Tree views, refresh and more button" are not part of the monaco editor."

**Comment by Cheneric95** [comment]
Azure Cloud Shell is actually using Monaco editor as a integrated editor. Storage creation issue should be resolved.

Please recreate and check again.

**Comment by fsteffi** [comment]
@Cheneric95 reopening this issue as per your comment

**Comment by fsteffi** [comment]
https://github.com/microsoft/vscode/issues/137717

**Comment by chrmarti** [comment]
@maertendMSFT Do you know where CloudShell issues can be reported?

**Comment by maertendMSFT** [comment]
This is our Github repo if you want to open something there as well: https://github.com/azure/cloudshell
Note this repo is just for our image, but we will still track the issue.

Additionally, @Cheneric95 is from the Cloud Shell team, so he can weigh in as well.

**Comment by chrmarti** [comment]
The Cloud Shell integration in the Azure Portal might be using an older version of the Tree widget. Make sure you update to the latest version of Monaco. (/cc @joaomoreno)

**Comment by chrmarti** [comment]
Closing stale issue. Let us know when you have additional info we can take action on. Thanks.
